templates/intro.html

```diff
@@ -21,7 +21,7 @@ <h1>Introduction to Longturn</h1>
 <h2>How to join</h2>
 <p>
 Longturn.net games are starting a few times per year in roughly regular intervals, although there is no rule. They are announced well ahead and registrations are usually open roughly a month ahead. To play a Longturn.net game you need to do three things:
-<p> 1. Install a Freeciv native client. This can be found at <href=http://freeciv.org>Freeciv.org</a>. Usually a game will be run on the newest version of server/client, but not always (for various reasons). Sometimes you may need to download and install an older version of the client. Older versions of Freeciv are also available on the Download page of Freeciv.org. Information about which Freeciv version is required for a particular LT game should be readily available on the game page (top right).
+<p> 1. Install the Freeciv21 client. This can be found at <href=https://github.com/longturn/freeciv21/releases>. Usually a game will be run on the latest stable version of server/client. Information about which Freeciv21 version is required for a particular LT game should be readily available on the game page (top right).
```

Comment:
Install Freeciv21 from [href]. Games are run on the latest stable release, however you can also play with the latest unstable release.

> We don't refer to the game as the "client", its just the game. We really don't worry about what version either. Technically the server is often latest alpha, but we don't need that on the website.